corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -45,6 +47,7 @@ hqDefine("data_dictionary/js/data_dictionary", [
         self.caseType = caseType;
         self.dataType = ko.observable(dataType);
         self.description = ko.observable(description);
+        self.fhirRsourcePropPath = ko.observable(fhirRsourcePropPath);
```

Comment:
Rsource => Resource

You're impressively consistent.

Reply:
thank you. I am :)

Reply:
Fixed.